cookies/templates/graph_coauthor_data.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% load breadcrumbs %}
+{% load addcss %}
+
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+    <a href="{% url 'index' %}">{% trans 'Home' %}</a>
+    {% breadcrumb_url 'Collections' 'collections' %}
+    {% breadcrumb_url 'Export' 'export-coauthor-data' %}
+</div>
+{% endblock %}
+
+{% block content %}
```

Comment:
See comment below -- we want to generate a GraphML file that the user can download.